app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  validates :name, presence: true
+  validates :posts_counter, minimum: 0
```

Comment:
- Good job done so far validating your models 👍 👍, but it seems you are using the wrong validation methods. Please check the [ruby guides](https://guides.rubyonrails.org/active_record_validations.html) for the proper validation methods 🙏